tfjs-core/scripts/test-integration.sh

```diff
@@ -26,32 +26,28 @@ echo 'Testing layers/converter/node/data against tfjs-core@master.'
 echo '######################'
 yarn build && yarn build-test-snippets && yarn yalc publish
 
-echo 'Cloning layers'
-git clone https://github.com/tensorflow/tfjs-layers.git --depth 1
+echo 'Testing layers'
 cd tfjs-layers
```

Comment:
I expected cd ../tfjs-layers (going to the parent first) but we’ll know for sure once you edit version.ts and let the CI run